app/src/main/java/com/keylesspalace/tusky/components/filters/FiltersAdapter.kt

```diff
@@ -0,0 +1,55 @@
+package com.keylesspalace.tusky.components.filters
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.databinding.ItemRemovableBinding
+import com.keylesspalace.tusky.entity.Filter
+import com.keylesspalace.tusky.util.getRelativeTimeSpanString
+
+class FiltersAdapter(context: Context, val listener: FiltersListener, filters: List<Filter>) :
+    ArrayAdapter<Filter>(context, R.layout.item_removable, filters) {
```

Comment:
Can we use RecyclerView and RecyclerViewAdapter instead please? ListView/ArrayAdapter are really old apis and with RecyclerView we would also get nice animations when an element gets deleted.